sites/public/lib/helpers.tsx

```diff
@@ -297,7 +297,7 @@ export const getUnitGroupSummary = (listing: Listing): UnitSummaryTable => {
     if (rentRange && rentAsPercentIncomeRange) {
       rent = (
         <>
-          {rentRange}, {rentAsPercentIncomeRange}
+          {rentRange},&nbsp;{rentAsPercentIncomeRange}
```

Comment:
`&nbsp;` isn't always consistent amongst browsers, it would be great to see a template literal here instead